docs/examples/use_cases/paddle/ssd/train.py

```diff
@@ -23,94 +23,82 @@
-            device="cpu",
-            aspect_ratio=[0.5, 2.0],
-            thresholds=[0, 0.1, 0.3, 0.5, 0.7, 0.9],
-            scaling=[0.3, 1.0],
-            bbox_layout="xyXY",
-            allow_no_crop=True,
-            num_attempts=50)
-        self.bbflip = ops.BbFlip(device="cpu", ltrb=True)
-
-        self.roi_decode = ops.ImageDecoderSlice(device="mixed")
-        self.resize = ops.Resize(
-            device="gpu",
-            resize_x=300,
-            resize_y=300,
-            min_filter=types.DALIInterpType.INTERP_TRIANGULAR)
-        self.hsv = ops.Hsv(device="gpu", dtype=types.FLOAT)  # use float to avoid clipping and
+    pipeline = Pipeline(batch_size, num_threads,
+                        local_rank, seed=42 + device_id)
+
+    with pipeline:
+        images, bboxes, labels = fn.coco_reader(file_root=file_root,
+                                                annotations_file=annotations_file,
+                                                skip_empty=True,
+                                                shard_id=local_rank,
+                                                num_shards=world_size,
+                                                ratio=True,
+                                                ltrb=True,
+                                                random_shuffle=False,
+                                                shuffle_after_epoch=True,
+                                                name="Reader")
+
+        crop_begin, crop_size, bboxes, labels = fn.random_bbox_crop(bboxes, labels,
+                                                                    device="cpu",
+                                                                    aspect_ratio=[
+                                                                        0.5, 2.0],
+                                                                    thresholds=[
+                                                                        0, 0.1, 0.3, 0.5, 0.7, 0.9],
+                                                                    scaling=[
+                                                                        0.3, 1.0],
+                                                                    bbox_layout="xyXY",
+                                                                    allow_no_crop=True,
+                                                                    num_attempts=1)
+        images = fn.image_decoder_slice(
+            images, crop_begin, crop_size, device="cpu", output_type=types.RGB)
+        flip_coin = fn.coin_flip(probability=0.5)
+        images = images.gpu()
+        images = fn.resize(images,
+                            resize_x=300,
+                            resize_y=300,
+                            min_filter=types.DALIInterpType.INTERP_TRIANGULAR)
+
+        saturation = fn.uniform(range=[0.5, 1.5])
+        contrast = fn.uniform(range=[0.5, 1.5])
+        brightness = fn.uniform(range=[0.875, 1.125])
```

Comment:
Just an opinion: I think it would be clearer to have these inlined in the operator call.

Reply:
Done